File: semester I/Praktikum Pemrograman I/Modul5_JovanGilbertNatamasindah_2310817310002/C/PRAK504-2310817310002-JovanGilbertNatamasindah.c
#include <stdio.h>

int reverse(int bilangan){

    int reversed = 0;

    while(bilangan != 0) {

        reversed = (reversed) * 10 + (bilangan % 10) ;
        bilangan /= 10;

    }

    return reversed;

}

int main() {

    int A, B;

    printf("Masukkan bilangan : ");
    scanf("%d %d",&A,&B);
    
    A = reverse(A);
    B = reverse(B);

    int C = A+B;
    printf("Hasilnya adalah : %d",reverse(C));

}

File: semester I/Praktikum Pemrograman I/Modul5_JovanGilbertNatamasindah_2310817310002/Python/PRAK504-2310817310002-JovanGilbertNatamasindah.py
def reverse(bilangan) :

    reversed = 0

    while(bilangan != 0) :

        reversed = (reversed) * 10 + (bilangan % 10)
        bilangan //= 10

    return reversed


A,B = map(int,input("Masukkan bilangan : ").split(" "))

A = reverse(A)
B = reverse(B)

C = A + B

print(f"Hasilnya adalah : {reverse(C)}")



